common/src/main/kotlin/org/waste/of/time/storage/cache/HotCache.kt

```diff
@@ -65,10 +67,19 @@ object HotCache {
      *
      * @param chunkX The X coordinate of the chunk.
      * @param chunkZ The Z coordinate of the chunk.
+     * @param dimension The dimension of the chunk.
      * @return True if the chunk is saved, false otherwise.
      */
     @Suppress("unused")
-    fun isChunkSaved(chunkX: Int, chunkZ: Int) = savedChunks.contains(ChunkPos.toLong(chunkX, chunkZ))
+    fun isChunkSaved(chunkX: Int, chunkZ: Int, dimension: RegistryKey<World>): Boolean {
```

Comment:
can either keep a method for the existing API without dimension and mark as deprecated, defaulting to current dimension.

or add a `breaks` clause for `xaeroplus < <version>` to the fmj

otherwise it will crash players who don't update both xp and worldtools at the same time